package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "MDN",
```

Comment:
Could you name that kuma as that's the name of the software while MDN is the product?